src/SFA.DAS.ApplyService.Web/Services/RoatpTaskListWorkflowService.cs

```diff
@@ -302,8 +303,14 @@ private string WhosInControlSectionStatus(Guid applicationId, IEnumerable<Applic
                     return TaskListSectionStatus.Blank;
                 }
 
-                var companiesHouseVerified = organisationVerificationStatus.CompaniesHouseDataConfirmed 
-                                         || organisationVerificationStatus.CompaniesHouseManualEntry;
+
+                if ((organisationVerificationStatus.CompaniesHouseManualEntry || organisationVerificationStatus.CharityCommissionManualEntry) 
+                    && !organisationVerificationStatus.CharityCommissionDataConfirmed
+                    && !organisationVerificationStatus.CompaniesHouseDataConfirmed
+                    && !organisationVerificationStatus.WhosInControlConfirmed)
+                    return TaskListSectionStatus.Next;
```

Comment:
Can you confirm this says In Progress when you're part way through entering Directors/Trustees

Reply:
You can't be part-way through.  The flow is now you enter one and it's confimed, there's no confirm at the end,, so there is no such thing as 'part way'.  If you have entered one, you can finish at that point and you are 'complete'.  The wording 'confirm and save' should have been changed to something like 'continue and save' a long time ago, as we do with management hierarchy.  I can't rmember why this was never done.

Reply:
That's current, accepted flow btw :+1:

Reply:
Yes, in management hierarchy, as you add each one, it's 'confirm and continue', and the list page is just 'continue'..  People in control elements are for each one you enter 'confirm and continue', but the list page also says 'confirm and continue'.  I remember arguing long ago it should say 'continue' to be clearer to the user, but it never got changed.

Reply:
This means there's no concept of 'in progress' for these items, they're either done (1 or more) or not done/next (0)

Reply:
sounds good 👍